### PR TITLE
Add new config for bumwings keyboard, based on RP2040 MCU

### PR DESCRIPTION
**Pull request by tubbytwins**
I'd like to add a new config and layouts for my "bumwings" keyboard design.

## Description

New keyboard config and layout

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

(none)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bumwings/keymaps/colemak-dh/keymap.c [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+     *  └───┘   │Hom│PDn│End│ │   ││   │   │Ret││   │ │ L │Dn │ R │   └───┘
+     *          └───┴───┴───┘ └───┘└───┘   └───┘└───┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_14x4_bum_thumbs(
+    	KC_TAB,   KC_Q,    KC_W,    KC_F,    KC_P,   KC_B,                            KC_J,   KC_L,    KC_U,     KC_Y,    KC_SCLN,  KC_EQUAL,
+    	KC_LCTL,  KC_A,    KC_R,    KC_S,    KC_T,   KC_G,   KC_BSPC, KC_BSPC,        KC_M,   KC_N,    KC_E,     KC_I,    KC_O,     KC_MINUS,
+    	KC_LSFT,  KC_Z,    KC_X,    KC_C,    KC_D,   KC_V,   KC_LGUI, KC_LALT,        KC_K,   KC_H,    KC_COMMA, KC_DOT,  KC_QUOTE, KC_RSFT,
+    	KC_GRAVE, KC_HOME, KC_PGDN, KC_PGUP, KC_END, KC_SPC, MO(1),   LT(2,KC_ENTER), KC_SPC, KC_LEFT, KC_UP,    KC_DOWN, KC_RIGHT, KC_SLASH
+    ),
+    [1] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/colemak-dh/keymap.c:29]:
```suggestion
    [1] = LAYOUT(
```

File: keyboards/bumwings/keymaps/colemak-dh/keymap.c [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    	KC_ESC,  KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                   HEX_0X,  KC_7, KC_8, KC_9, KC_SLASH, KC_PLUS,
+    	_______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL,  KC_BSPC, KC_DOT,  KC_4, KC_5, KC_6, KC_BSLS,  KC_UNDS,
+    	_______, KC_COLN, KC_LCBR, KC_RCBR, KC_LBRC, KC_RBRC, KC_LGUI, KC_LALT, KC_0,    KC_1, KC_2, KC_3, KC_PIPE,  KC_RSFT,
+    	KC_TILD, _______, _______, _______, _______, _______, _______, KC_PENT, _______, _______, _______, _______, _______, KC_APP
+    ),
+    [2] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/colemak-dh/keymap.c:35]:
```suggestion
    [2] = LAYOUT(
```

File: keyboards/bumwings/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Matthew A. Schnoor (@tubbytwins)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include "bumwings_common.h"
```
Comment (drashna) [inline keyboards/bumwings/keymaps/default/keymap.c:6]:
```suggestion
```

File: keyboards/bumwings/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+     * ├───┤ A ├───┼───┼───┼───┼───┘           └───┼───┼───┼───┼───┤ ; ├───┤
+     * │Ctl├───┤ X │ C │ V │ B │                   │ N │ M │ , │ . ├───┤ - │
+     * ├───┤ Z ├───┴───┴───┴───┘   ┌───┐   ┌───┐   └───┴───┴───┴───┤ ' ├───┤
+     * │Shf├───┘    ┌───┐     ┌───┐│Gui│   │Alt│┌───┐     ┌───┐    └───┤Shf│
+     * └┬──┴┐       │PUp│     │   │├───┤   ├───┤│   │     │Up │       ┌┴──┬┘
+     *  │ ` │   ┌───┼───┼───┐ │Spc││l1 │   │l2,││Spc│ ┌───┼───┼───┐   │ / │
+     *  └───┘   │Hom│PDn│End│ │   ││   │   │Ret││   │ │ L │Dn │ R │   └───┘
+     *          └───┴───┴───┘ └───┘└───┘   └───┘└───┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/default/keymap.c:23]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/bumwings/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    	KC_TAB,   KC_Q,    KC_W,    KC_E,    KC_R,   KC_T,                            KC_Y,   KC_U,    KC_I,     KC_O,    KC_P,     KC_EQUAL,
+    	KC_LCTL,  KC_A,    KC_S,    KC_D,    KC_F,   KC_G,   KC_BSPC, KC_BSPC,        KC_H,   KC_J,    KC_K,     KC_L,    KC_SCLN,  KC_MINUS,
+    	KC_LSFT,  KC_Z,    KC_X,    KC_C,    KC_V,   KC_B,   KC_LGUI, KC_LALT,        KC_N,   KC_M,    KC_COMMA, KC_DOT,  KC_QUOTE, KC_RSFT,
+    	KC_GRAVE, KC_HOME, KC_PGDN, KC_PGUP, KC_END, KC_SPC, MO(1),   LT(2,KC_ENTER), KC_SPC, KC_LEFT, KC_UP,    KC_DOWN, KC_RIGHT, KC_SLASH
+    ),
+    [1] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/default/keymap.c:29]:
```suggestion
    [1] = LAYOUT(
```

File: keyboards/bumwings/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    	KC_ESC,  KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                   HEX_0X,  KC_7, KC_8, KC_9, KC_SLASH, KC_PLUS,
+    	_______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL,  KC_BSPC, KC_DOT,  KC_4, KC_5, KC_6, KC_BSLS,  KC_UNDS,
+    	_______, KC_COLN, KC_LCBR, KC_RCBR, KC_LBRC, KC_RBRC, KC_LGUI, KC_LALT, KC_0,    KC_1, KC_2, KC_3, KC_PIPE,  KC_RSFT,
+    	KC_TILD, _______, _______, _______, _______, _______, _______, KC_PENT, _______, _______, _______, _______, _______, KC_APP
+    ),
+    [2] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/default/keymap.c:35]:
```suggestion
    [2] = LAYOUT(
```

File: keyboards/bumwings/keymaps/dvorak/keymap.c [hunk 7]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Matthew A. Schnoor (@tubbytwins)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include "bumwings_common.h"
```
Comment (drashna) [inline keyboards/bumwings/keymaps/dvorak/keymap.c:6]:
```suggestion
```

File: keyboards/bumwings/keymaps/dvorak/keymap.c [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+     * ├───┤ A ├───┼───┼───┼───┼───┘           └───┼───┼───┼───┼───┤ S ├───┤
+     * │Ctl├───┤ Q │ J │ K │ X │                   │ B │ M │ W │ V ├───┤ - │
+     * ├───┤ ; ├───┴───┴───┴───┘   ┌───┐   ┌───┐   └───┴───┴───┴───┤ Z ├───┤
+     * │Shf├───┘    ┌───┐     ┌───┐│Gui│   │Alt│┌───┐     ┌───┐    └───┤Shf│
+     * └┬──┴┐       │PUp│     │   │├───┤   ├───┤│   │     │Up │       ┌┴──┬┘
+     *  │ ` │   ┌───┼───┼───┐ │Spc││l1 │   │l2,││Spc│ ┌───┼───┼───┐   │ / │
+     *  └───┘   │Hom│PDn│End│ │   ││   │   │Ret││   │ │ L │Dn │ R │   └───┘
+     *          └───┴───┴───┘ └───┘└───┘   └───┘└───┘ └───┴───┴───┘
+     */
+    [0] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/dvorak/keymap.c:23]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/bumwings/keymaps/dvorak/keymap.c [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    	KC_TAB,   KC_QUOTE, KC_COMMA, KC_DOT,  KC_P,   KC_Y,                            KC_F,   KC_G,    KC_C, KC_R, KC_L, KC_EQUAL,
+    	KC_LCTL,  KC_A,     KC_O,     KC_E,    KC_U,   KC_I,   KC_BSPC, KC_BSPC,        KC_D,   KC_H,    KC_T, KC_N, KC_S, KC_MINUS,
+    	KC_LSFT,  KC_SCLN,  KC_Q,     KC_J,    KC_K,   KC_X,   KC_LGUI, KC_LALT,        KC_B,   KC_M,    KC_W, KC_V, KC_Z, KC_RSFT,
+    	KC_GRAVE, KC_HOME,  KC_PGDN,  KC_PGUP, KC_END, KC_SPC, MO(1),   LT(2,KC_ENTER), KC_SPC, KC_LEFT, KC_UP, KC_DOWN, KC_RIGHT, KC_SLASH
+    ),
+    [1] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/dvorak/keymap.c:29]:
```suggestion
    [1] = LAYOUT(
```

File: keyboards/bumwings/keymaps/dvorak/keymap.c [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+    	KC_ESC,  KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                   HEX_0X,  KC_7, KC_8, KC_9, KC_SLASH, KC_PLUS,
+    	_______, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_DEL,  KC_BSPC, KC_DOT,  KC_4, KC_5, KC_6, KC_BSLS,  KC_UNDS,
+    	_______, KC_COLN, KC_LCBR, KC_RCBR, KC_LBRC, KC_RBRC, KC_LGUI, KC_LALT, KC_0,    KC_1, KC_2, KC_3, KC_PIPE,  KC_RSFT,
+    	KC_TILD, _______, _______, _______, _______, _______, _______, KC_PENT, _______, _______, _______, _______, _______, KC_APP
+    ),
+    [2] = LAYOUT_14x4_bum_thumbs(
```
Comment (drashna) [inline keyboards/bumwings/keymaps/dvorak/keymap.c:35]:
```suggestion
    [2] = LAYOUT(
```

File: keyboards/bumwings/readme.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+# BumWings
+
+![BumWings](https://i.imgur.com/WwpVsNN.jpg)
```
Comment (drashna) [inline keyboards/bumwings/readme.md:3]:
```suggestion
![BumWings](https://i.imgur.com/WwpVsNNh.jpg)
```

File: keyboards/bumwings/rules.mk [hunk 12]
```diff
@@ -0,0 +1,6 @@
+# Yup
+
+CAPS_WORD_ENABLE = yes
+
+SRC += bumwings_common.c
```
Comment (drashna) [inline keyboards/bumwings/rules.mk:5]: When properly named, this isn't needed. 
```suggestion
```

File: keyboards/bumwings/v001/info.json [hunk 13]
```diff
@@ -0,0 +1,80 @@
+{
```
Comment (drashna) [inline keyboards/bumwings/v001/info.json:1]: this can be merged with the top level,

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bumwings/config.h [hunk 14]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2022 Matthew A. Schnoor (@tubbytwins)
```
Comment (fauxpark) [inline keyboards/bumwings/config.h:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**